dmt/templates/main/project_detail.html

```diff
@@ -321,13 +328,25 @@ <h4>
   </div>
 
   <div class="tab-pane fade" id="personnel">
-    <ul>
+    <h4>Current</h4>
+    <br />
```

Comment:
In general - `<br />` tags aren't necessary. Spacing is better handled with CSS. No big deal though - I can still merge this.

Reply:
Agreed. I was doing the front-end layout quickly & took the shortcut.